aries-site/src/pages/templates/selector.mdx

```diff
@@ -0,0 +1,93 @@
+    'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-core/src/js/components/core/Selector/Selector.js',
+    'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-core/src/js/components/core/Selector/SelectorGroup.js',
+  ]}
+  github="https://github.com/grommet/hpe-design-system/tree/master/aries-core/src/js/components/core/Selector"
+  width="100%"
+>
+  <SimpleSelector />
+</Example>
+
+## Use cases
+
+### Quick Filtering
+
+Enables users to filter data based on high-level visual options to narrow down datasets within tables and list. Making it quickly recognizable, based on specific criteria, enhancing efficiency and usability.
+
+<Example
+  figma="TBD"
+  code={[
+    'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/selector/QuickFilter.js',
+    'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-core/src/js/components/core/Selector/Selector.js',
+    'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-core/src/js/components/core/Selector/SelectorGroup.js',
+  ]}
+  github="https://github.com/grommet/hpe-design-system/tree/master/aries-core/src/js/components/core/Selector"
+  width="100%"
+>
+  <QuickFilter />
+</Example>
+
+### Quick option selection
+
+Helps users quickly navigate through choices, ensuring they find the relevant options with ease.
+
+<Example
+  code=""
+  height={{ min: 'small' }}
+  width="100%"
+  code={[
+    'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/selector/SupportSelector.js',
+  ]}
+  github="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/selector/SupportSelector.js"
+>
+  <SupportSelector />
+</Example>
+
+## Anatomy
+
+
+| Label | Region                  | Purpose                                                      | Required | Notes                                                   |
+| :---: | --------------------    | ------------------------------------------------------------ | :------: | ------------------------------------------------------- |
+| **1** | **Title**               | Heading providing context to the options available.          | Optional | Titles should use a Heading of the correct semantic heading level based on where the selector is in the document. |
+| **2** | **Description**         | Brief explanation or additional information about the choices.| Optional |    --                                                  |
+| **3** | **Counter of Items**    | Displays number of items available for selection.            | Optional | -- |
+| **4** | **Icon**                | Visual aid to represent the category or type of selection.   | Optional | a11yTitle required. |
+| **5** | **Selector Indicator**  | Visual cue (such as a checkmark or highlight) to denote selected items. | Optional | aria-label required. |
+
+## Types
+
```

Comment:
I would suggest we use a different visual representation for "Types", like a table with a visual and an explanation, so the user can easily understand the difference not only on functionality but on type of component.

Also, if we are delivering all the examples on the TEMP kitchen sink, we might need to amplify this section, since we have different visual types for single select and multi select.

Reply:
I'm not sure I'm following. For types we don't include imagery but rather indicate the "types" of selectors there are.

For kitchen sink, these are all TBD and will be clarified soon. :)

Reply:
Kitchen sink is just for us devs to test will not be published

Reply:
@julauxen Im also not sure I follow your visual with "types"